README.md

```diff
@@ -1,68 +1,206 @@
 # USACE Flood Modeling Geoprocessing
 
-A [Spark Job Server](https://github.com/spark-jobserver/spark-jobserver) job for USACE flood modeling.
+This repository contains two projects for facilitating the [USACE Flood Model web app](https://github.com/azavea/usace-flood-model):
 
-## Usage
+ 1. [`ingest`](#ingest): For taking input elevation data and processing it into a format usable by [GeoTrellis](https://github.com/geotrellis/geotrellis)
+ 2. [`server`](#server): A [Spray](https://github.com/spray/spray) based web service that performs geoprocessing operations
 
-First, build the assembly JAR for this project:
+Both projects are run using [Apache Spark](http://spark.apache.org/).
+
+## `ingest`
+
+This project takes input elevation data and processes it into RDDs that will be ready by GeoTrellis. The input data was originally in DEM files supplied by the client. Using [GDal](http://www.gdal.org/), these were stitched together, then clipped to the relevant area of interest, and tiled (see [azavea/usace-flood-model#2](https://github.com/azavea/usace-flood-model/issues/2)).
+
+The input data is in EPSG:4269. The `server` provides two main services: calculation of inundated percent area based on a given polygon, and rendering the corresponding tiles. The area calculation must be done in the original projection, while rendering must be done using WebMercator. Thus, we have two tile sets, one in each projection.
+
+### Build
+
+To build the ingest JAR, execute the following:
 
 ```bash
-$ git clone https://github.com/azavea/usace-flood-geoprocessing.git
-$ cd usace-flood-geoprocessing
-$ ./sbt assembly
+$ ./sbt "project ingest" assembly
 ```
 
-To use a Docker based Scala build environment, you can use:
+which should result in `ingest/target/scala-2.10/usaceflood-ingest-assembly-0.1.0-SNAPSHOT.jar`.
```

Comment:
Ah, thanks! Will update. We will probably also have to update here: https://github.com/azavea/usace-flood-model/blob/develop/deployment/ansible/roles/usace.geoprocessing/defaults/main.yml#L4